src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -444,7 +444,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         OpenSettingsArgs() = default;
         OpenSettingsArgs(const SettingsTarget& target) :
             _Target{ target } {}
-        GETSET_PROPERTY(SettingsTarget, Target, SettingsTarget::SettingsFile);
+        GETSET_PROPERTY(SettingsTarget, Target, SettingsTarget::SettingsUI);

```

Comment:
last q: should we leave this the same and only update the `target` in defaults.json? I'd like to make sure we don't break people who added their own openSettings key bindings.

Reply:
You're right. But I don't like that you're right hahaha. Begrudgingly making the change :P

Reply:
On a more serious note, should we consider this a breaking change we want to commit to for v2? Or for stable when SUI goes to stable?

Reply:
It costs us nothing to let users have their old bindings work the way they always have, right? Here's my thinking. If a user _bound settings to open_, IN their JSON file, that's probably what they are expecting.

Reply:
That's fair. And when the actual Actions page goes in, the user will probably have to explicitly choose what they want to open. So this seems fine :)

Reply:
Gah I hate to come in here and disagree after this has merged, but I think moving it to `SettingsTarget::SettingsUI` makes sense. I know we're going to be breaking people now, but a year from now I feel like we would have wished we moved the target to the settings UI.

Reply:
We can make the decision to break peoples' settings in the future.